### PR TITLE
Add Site affiliation post

### PR DESCRIPTION
**Pull request by mihajlija**
Changes proposed in this pull request:

- Add Site affiliation post
- Add @mihajlija as dcc author

### CODE REVIEW

**Review by robdodson — APPROVED** [review]

File: site/en/blog/site-affiliation/index.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+    specify the string `delegate_permission/common.get_login_creds`.  
+    The `target` field is an object that specifies the asset the declaration
+    applies to. The following fields identify a website:
+    
+    <br>
+    
+    <table>
+      <tr>
+        <td><code>namespace</code></td>
+        <td>Must be web for websites.</td>
```
Comment (agektmr) [inline site/en/blog/site-affiliation/index.md:107]:
```suggestion
        <td>Must be `web` for websites.</td>
```

File: site/en/blog/site-affiliation/index.md [hunk 2]
```diff
@@ -0,0 +1,221 @@
...
+        <code>https://<strong>domain</strong>[:<strong>optional_port</strong>]</code>;
+        for example, <code>https://www.example.com</code>.</td>
+      </tr>
+    </table>
+
+    See the
+    [Digital Asset Links reference](https://developers.google.com/digital-asset-links/v1/statements)
+    for details.
+
+1.  Host the Digital Assets Link JSON file at the following location on
```
Comment (agektmr) [inline site/en/blog/site-affiliation/index.md:121]:
```suggestion
1.  Host the Digital Asset Links JSON file at the following location on
```

File: site/en/blog/site-affiliation/index.md [hunk 3]
```diff
@@ -0,0 +1,221 @@
...
+    the sign-in domain: `https://domain[:optional_port]/.well-known/assetlinks.json`.
+
+    <br>
+
+    In this example, the domain is `www.example.com`, so the JSON file should
+    be hosted at `https://www.example.com/.well-known/assetlinks.json`.   
+
+    <br>
+
+    The MIME type for the Digital Assets Link file needs to be JSON. Make sure
```
Comment (agektmr) [inline site/en/blog/site-affiliation/index.md:131]:
```suggestion
    The MIME type for the Digital Asset Links file needs to be JSON. Make sure
```

File: site/en/blog/site-affiliation/index.md [hunk 4]
```diff
@@ -0,0 +1,221 @@
...
+      "relation": ["delegate_permission/common.get_login_creds"],
+      "target": {
+        "namespace": "web",
+        "site": "https://www.example.co.uk"
+      }
+     }]
+     ```
+
+1.  Ensure that your host permits Google to retrieve your Digital Asset
+    Link file. If you have a `robots.txt` file, it must allow the Googlebot
```
Comment (agektmr) [inline site/en/blog/site-affiliation/index.md:158]:
```suggestion
    Links file. If you have a `robots.txt` file, it must allow the Googlebot
```

File: site/en/blog/site-affiliation/index.md [hunk 5]
```diff
@@ -0,0 +1,221 @@
...
+
+{% Aside %}
+Be aware that once you set up a DAL file, it may take a while for Googlebot to fetch the
+resource and recognize the association.
+{% endAside %}
+
+## Associate multiple websites with one another
+
+You can associate multiple websites with one another by specifying each one in
+the Digital Assets Link file. For example, to associate the `example.com`,
```
Comment (agektmr) [inline site/en/blog/site-affiliation/index.md:178]:
```suggestion
the Digital Asset Links file. For example, to associate the `example.com`,
```

File: site/en/blog/site-affiliation/index.md [hunk 6]
```diff
@@ -0,0 +1,221 @@
...
+     ],  
+     "target":{  
+        "site":"https://www.example.co.jp",  
+        "namespace":"web"  
+     }  
+  }]
+```
+
+{% Aside %}
+You can use include statements if you need to issue statements from
```
Comment (agektmr) [inline site/en/blog/site-affiliation/index.md:214]:
```suggestion
You can use [include statements](https://developers.google.com/digital-asset-links/v1/statements#scaling-to-dozens-of-statements-or-more) if you need to issue statements from
```

**Review by A1T0M3CK — COMMENTED** [review]
0-=-0

**Pull request merged**